--- packages/JEGA/SOGA/src/SOGA.cpp	2014-06-03 17:34:06.255994118 -0500
+++ packages/JEGA/SOGA/src/SOGA-new.cpp	2014-06-03 17:34:41.449329229 -0500
@@ -352,7 +352,7 @@
 
     // now flush out all non-optimal designs.
     pair<double, vector<DesignOFSortSet::const_iterator> > bests(
-        SingleObjectiveStatistician::FindMinSumDesigns(
+        SingleObjectiveStatistician::FindMinSumMinViolateDesigns(
             pop.GetOFSortContainer(), this->GetWeights()
             )
         );
